template.yaml

```diff
@@ -127,21 +140,67 @@ Resources:
           IS_LOCAL: false
           KBV_SESSION_TABLE_NAME: !Ref KBVSessionTable
           POWERTOOLS_SERVICE_NAME: di-ipv-cri-kbv-api
-          EXPERIAN_API_WRAPPER_SAA_RESOURCE: http://localhost:8080/question-request
-          EXPERIAN_API_WRAPPER_RTQ_RESOURCE: http://localhost:8080/question-answer
-          EXPERIAN_API_WRAPPER_URL: http://localhost:8080/
+      Events:
+        KBVSessionFunctionApi:
+          Type: Api
+          Properties:
+            Path: /session
+            Method: post
+            RestApiId: !Ref KBVSessionApi
+
+  KBVQuestionFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./lambdas/question
+      Handler: uk.gov.di.ipv.cri.kbv.api.handler.QuestionHandler::handleRequest
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - DynamoDBReadPolicy:
+            TableName: !Ref KBVSessionTable
+        - DynamoDBWritePolicy:
+            TableName: !Ref KBVSessionTable
+      Runtime: java11
+      MemorySize: 512
+      Tracing: Active
+      Environment:
+        Variables:
+          IS_LOCAL: false
+          KBV_SESSION_TABLE_NAME: !Ref KBVSessionTable
+          POWERTOOLS_SERVICE_NAME: di-ipv-cri-kbv-api
+          EXPERIAN_API_WRAPPER_SAA_RESOURCE: /question-request
+          EXPERIAN_API_WRAPPER_URL: https://u7tpgfw3bh.execute-api.eu-west-2.amazonaws.com/dev
+
```

Comment:
can we remove this newline to join the event to the Function pls?

Reply:
Done.